C/services/south/south.cpp

```diff
@@ -356,6 +374,7 @@ void SouthService::start(string& coreAddress, unsigned short corePort)
 						pollCount += (int) vec->size();
 						delete vec; 	// each reading object inside vector has been allocated on heap and moved to Ingest class's internal queue
 					}
+					throttlePoll();
```

Comment:
This would be called on each poll interval? That could be a lot for higher rates. May be we can call this less frequently? May be every 5 seconds (Greatest common divisor of SOUTH_THROTTLE_DOWN_INTERVAL and SOUTH_THROTTLE_UP_INTERVAL)